### ISSUE TITLE
Crashed after using F8 to cycle through a bunch of symbols and Ctrl+V replace them

### ISSUE THREAD

**Issue by samswain** [slynch8/10x#345]
Searched for some "TSharedPtr<MyStruct>" entries.
Cycling through them with F8, replacing each one with "MyStruct::Ptr" (paste Ctrl+V).
Did most of them, then crashed on F8.

**Comment by samswain** [comment]
Attached the crashdump (had to zip it up as Github doesn't seem to recognise gzip files).
(https://github.com/slynch8/10x/files/6946535/10x_crash_2021_8_6_16_34_28.dmp.gzip.zip)

**Comment by slynch8** [comment]
This is a nasty one, looks like a memory corruption, but can't tell much from the minidump. Are you able to reproduce it?

**Comment by samswain** [comment]
I'm not sure, I did re-do the same changes, but I saved every few :/  There were about two dozen places I needed to change the shared ptr.

**Comment by samswain** [comment]
Might not be related, but the end of one of the files seems to have ended up corrupted.  Could be on the restore of unsaved files (although I don't think it restored all the ones I had edited as I went).


```
// EPILOGUE
#undef LOCTEXT_NAMESPACE

#undef LOCTEXT_NAMESPACE

#undef LOCTEXT_NAMESPACE

ACE

LOCTEXT_NAMESPACE
```

**Comment by samswain** [comment]
Another one here:

```
// EPILOGUE
#undef LOCTEXT_NAMESPACE

ACE
```

**Comment by slynch8** [comment]
That's really bad, sorry about that. When it restores a backup file you can undo the restore.

Trying to repro here, was the F8 after a Workspace Search?

**Comment by samswain** [comment]
No worries.  Yes, Alt+F for the text.  Just manually 'renaming' something, each step being: F8, Cursorkeys, Ctrl+Shift+Cursorkeys to highlight the text, Ctrl+V to paste the new version in, repeat.

**Comment by slynch8** [comment]
I've got the feeling this is going to be a tricky one to track down. The corruption probably happened earlier so may not be related to the last operation. Would you mind turning on full crash dumps? Setting Misc.CrashDumpMode: Full.

**Comment by samswain** [comment]
Sure. Happy to provide more info. Will post here if it happens again (one day).

**Comment by slynch8** [comment]
I think I've fixed it (0.28.0). I found a string memory corruption in the code for adding new lines to a comment blocks.
